lib/server.js

```diff
@@ -137,7 +137,6 @@ function handleResponse(opts, req, resp, response) {
         }
         opts.log(logLevel, {
             message: response.message,
-            root_req: req,
```

Comment:
This was actually overwriting the `root_req` property we've set when we created this child logger and bypasses the header whitelist we've set up.